app/javascript/gobierto_visualizations/webapp/containers/contracts/ContractsShow.vue

```diff
@@ -132,6 +237,40 @@ export default {
       this.contract_type = contract_type
       this.permalink = permalink
       this.assignee_routing_id = assignee_routing_id
+      this.contractor = contractor
+      this.contract_type = contract_type
+      this.start_date = start_date
+      this.end_date = end_date
+      this.award_date = award_date
+      this.batch_number = +batch_number
+      this.minor_contract = minor_contract
+      this.open_proposals_date = open_proposals_date || null
+      this.submission_date = submission_date || null
+      this.cpvs = cpvs
+      this.category_title = category_title
+    }
+
+    if (this.hasBatch) this.groupBatchs()
```

Comment:
The plural of batch is batches, not batchs

Reply:
Done!